### DISCUSSION TITLE
Open Source "Issue" releated doubt

### DISCUSSION THREAD

**Discussion by Sisir-kumar-Das** [piyushgarg-dev/piyushgargdev-nextjs#356] [Q&A]
Hey, I am SIsir. I am begineer in open Source contribution, Can anyone clarify my doubt releted to "Issue" that One can create new issue and the same person can also fix that issue or not?

**Comment by Palakiew** [comment]
Hey there! In open source projects, it's generally encouraged for anyone to create new issues, regardless of whether they plan to fix them or not. Creating an issue helps bring attention to a problem or improvement that needs to be addressed. While it's not necessary for the same person to fix the issue they created, they are more than welcome to contribute a fix if they have the skills and knowledge to do so. Open source is all about collaboration, so multiple people can work together to solve issues. Don't hesitate to ask if you have any more questions about open source contribution! 😊🌟
